CONTRIBUTING_PACKAGES.md

```diff
@@ -150,6 +150,6 @@ Additionally, Great Expectations adds the following tags to indicate issue statu
 
 - The`help wanted` tag identifies useful issues that require help from community contributors to accelerate development.
 
-- The `enhacement` and `expectation-request` tags identify new Great Expectations features that require additional investigation and discussion. 
+- The `enhacement` and `expectation-request` tags identify new Great Expectations features that require additional investigation and discussion.
```

Comment:
It is not clear why this text appears as changed -- it seems identical to what was there before, save for the blank space at the end (result of linter).